packages/react-native/sdks/hermes-engine/hermes-utils.rb

```diff
@@ -233,6 +235,27 @@ def resolve_url_redirects(url)
     return (`curl -Ls -o /dev/null -w %{url_effective} \"#{url}\"`)
 end
 
+# react-native-macos does not publish macos specific hermes artifacts
+# so we attempt to find the latest patch version of the iOS artifacts and use that
+def findLastestVersionWithArtifact(version)
```

Comment:
Same comment about adding diff tags to method.

Reply:
fixed